charts/k3s/templates/statefulset.yaml

```diff
@@ -143,9 +143,9 @@ spec:
       {{- if .Values.priorityClassName }}
       priorityClassName: {{ .Values.priorityClassName }}
       {{- end }}
-      {{- if .Values.fsGroup }}
+      {{- if .Values.securityContext }}
       securityContext:
-        fsGroup: {{ .Values.fsGroup }}
```

Comment:
This doesn't look like backward compatible change. I would postpone this until a minor release or make it backward-compatible. Same ofc for the k0s.